base_wamas_ubl/lib/wamas/utils.py

```diff
@@ -211,9 +211,16 @@ def get_date_from_field(*args):
     return res
 
 
-def convert_unit_code(key, val):
+def convert_unit_code(key, val, extra_data=False):
```

Comment:
why not an explicit  key?

```suggestion
def convert_unit_code(key, val, unit_mapping=None):
```